### ISSUE TITLE
[CMIP6 CMOR-ization & ESGF-publication] NorESM2-LM - piClim-2xVOC

### ISSUE THREAD

**Issue by DirkOlivie** [NorESMhub/noresm2cmor#75]
### Mandatory information:

**Full path to the case(s) of the experiment on NIRD**
/projects/projects/NS9560K/noresm/cases

**experiment_id**
piClim-2xVOC

**model_id**
NorESM2-LM

**CASENAME(s) and years to be CMORized**
NF1850norbc2voc_f19_20190805, 1-30

### Optional information

**parent_experiment_id**
piControl

**parent_experiment_rip**
r1i1p1f1

**parent_time_units**
'days since 0421-01-01'

**branch_method**
'Hybrid-restart from year 1751-01-01 of piControl',

**other information**

**Comment by YanchunHe** [comment]
Experiment(s) ready to be published to ESGF:
---

**data path**
- /projects/NS9034K/CMIP6/AerChemMIP/NCC/NorESM2-LM/piClim-2xVOC

**Version**
- v20190920

**other information**
- PrePARE quality-check, passed
- sha256sum, calculated

**Note**
The required source_type (model componentes) are "AGCM AER CHEM ",
but during CMORziation, this is set as "AGCM AER ".
Therefore, there is a warming during cmorization, and the cmorized data have atrribute 'source_type" as "AGCM AER " instead of as required "AGCM AER CHEM "
But I don't think this is an real issue and will not affect the data itself at all.
So I did not redo the cmorization to correct this very trivial  warning.

**Comment by monsieuralok** [comment]
@DirkOlivie @YanchunHe published

**Comment by YanchunHe** [comment]
cmorized with additional fields

**data path**
- /tos-project1/NS9034K/CMIP6/AerChemMIP/NCC/NorESM2-LM/piClim-2xVOC

**version**
- v20191108

**sha256sum**
- .r1i1p1f1.sha256sum_v20191108b

**Comment by monsieuralok** [comment]
@DirkOlivie @YanchunHe published

**Comment by YanchunHe** [comment]
Cmorized with additional iLAMB variables (#262), AERday `zg500` (#263) and corrected `fNup` (#251).

They are ready to be published to ESGF.

**data path**
- /projects/NS9034K/CMIP6/.cmorout/NorESM2-LM/piClim-2xVOC
- /projects/NS9034K/CMIP6/AerChemMIP/NCC/NorESM2-LM/piClim-2xVOC

**version**
- v20210203

**sha256sum**
/projects/NS9034K/CMIP6/AerChemMIP/NCC/NorESM2-LM/piClim-2xVOC
- .r1i1p1f1.sha256sum_v20210203

**Comment by monsieuralok** [comment]
same here should it be .r1i1p1f1.sha256sum_v20210203 or .r1i1p2f1.sha256sum_v20210203 ?

**Comment by YanchunHe** [comment]
> @YanchunHe same here should it be .r1i1p1f1.sha256sum_v20210203 or .r1i1p2f1.sha256sum_v20210203 ?

should be .r1i1p1f1.sha256sum_v20210203

There is another separate issue for .r1i1p2f1.sha256sum_v20210203

**Comment by monsieuralok** [comment]
@YanchunHe then, .r1i1p1f1.sha256sum_v20210203 does not exists.

**Comment by YanchunHe** [comment]
Hi @adagj 
Could you check the permission of /projects/NS9560K/noresm/cases/NF1850norbc2voc_f19_20190805
Thanks!

**Comment by adagj** [comment]
Hi @YanchunHe 
Sorry for a very late reply!

I don't have access to  /projects/NS9560K/noresm/cases/NF1850norbc2voc_f19_20190805, only  /projects/NS9560K/noresm/cases/NF1850norbc2voc_f19_20191004.

**Comment by YanchunHe** [comment]
Hi @adagj 

That is a strange. I see you own these two folders:


````
$ ll |grep 'NF1850norbc2voc_f19_20190805'
79:drw-rw-r-- 11 adagj    ns9560k  4.0K Aug 19  2019 NF1850norbc2voc_f19_20190805/
$ ll |grep 'NF1850norbc2voc_f19_20191004'
80:drwxr-xr-x  8 adagj    ns9560k  4.0K Nov 19  2019 NF1850norbc2voc_f19_20191004/
```

**Comment by adagj** [comment]
@YanchunHe 
Yes, indeed.
[adagj@login2-nird-tos cases]$ ls NF1850norbc2voc_f19_20190805/
ls: cannot access NF1850norbc2voc_f19_20190805/logs: Permission denied
ls: cannot access NF1850norbc2voc_f19_20190805/cpl: Permission denied
ls: cannot access NF1850norbc2voc_f19_20190805/atm: Permission denied
ls: cannot access NF1850norbc2voc_f19_20190805/ocn: Permission denied
ls: cannot access NF1850norbc2voc_f19_20190805/lnd: Permission denied
ls: cannot access NF1850norbc2voc_f19_20190805/archive.log.190818-165542: Permission denied
ls: cannot access NF1850norbc2voc_f19_20190805/esp: Permission denied
ls: cannot access NF1850norbc2voc_f19_20190805/rof: Permission denied
ls: cannot access NF1850norbc2voc_f19_20190805/rest: Permission denied
ls: cannot access NF1850norbc2voc_f19_20190805/archive.log.190816-034345: Permission denied
ls: cannot access NF1850norbc2voc_f19_20190805/ice: Permission denied
ls: cannot access NF1850norbc2voc_f19_20190805/archive.log.190813-120252: Permission denied
archive.log.190813-120252  archive.log.190816-034345  archive.log.190818-165542  atm  cpl  esp  ice  lnd  logs  ocn  rest  rof

**Comment by YanchunHe** [comment]
I see, you probably need to add 'x' permission to your self.

try `chmod 755 NF1850norbc2voc_f19_20190805`

**Comment by adagj** [comment]
@YanchunHe 
Can you try and see if you have access now?

**Comment by YanchunHe** [comment]
> @YanchunHe
> Can you try and see if you have access now?

Yes, I do have access now. Thanks!

**Comment by YanchunHe** [comment]
New dataset version to fix issues #270, #271, #272 and #273 is ready to be published:

**data path**
- /projects/NS9034K/CMIP6/AerChemMIP/NCC/NorESM2-LM/piClim-2xVOC
- /projects/NS9034K/CMIP6/.cmorout/NorESM2-LM/piClim-2xVOC

**version**
- v20210319

**sha256sum**
- /projects/NS9034K/CMIP6/AerChemMIP/NCC/NorESM2-LM/piClim-2xVOC
- .r1i1p1f1.sha256sum_v20210319

**Comment by monsieuralok** [comment]
@YanchunHe published
